src/Hangfire.Tags/Storage/ITagsMonitoringApi.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Hangfire.Tags.Storage
 {
-    internal interface ITagsMonitoringApi
+    public interface ITagsMonitoringApi
```

Comment:
Not required for the connection leak issue

Reply:
yup, absolutely, it should be a different PR.